docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-is-used-on-the-project-level.md

```diff
@@ -14,11 +14,17 @@ related:
     link: docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-method-is-overridden-on-the-project-level.html
 ---
 
+Private API is used on the project level. The page describes why entities on the project side have to avoid private api using. The project has to follow the rule to be able to correctly receive Spryker autoupdates.
```

Comment:
```
Private API entities shouldn't be used on the project side. ...
```

Reply:
++

Replaced with the suggested structure